COLLABORATOR_GUIDE.md

```diff
@@ -222,6 +222,24 @@ not used in other CI test runs (such as tests in the `internet` or `pummel`
 directories). It can also make sure tests pass when provided with a flag not
 used in other CI test runs (such as `--worker`).
 
+##### citgm-smoker
```

Comment:
The instructions are sound but there's nothing mentioned here that are particular to the `citgm-smoker` job. The same parameters exist for the other jobs mentioned above (e.g. `node-test-commit-v8-linux`, `node-test-commit-custom-suites` (although the link needs updating, I'll PR that separately)).

The last paragraph applies to any job you start in Jenkins.

Reply:
Haven't used the other ones but `node-test-pull-request` is different.

> Copy/paste the URL for the job into a comment in the pull request.

The bot picks up `node-test-pull-request` and automatically posts it.

Reply:
> same parameters exist for the other jobs mentioned above

`node-stress-single-test` does not have `REBASE_ONTO`

Reply:
Updated, PTAL